src/gettingstarted/own-code/service-configuration.md

```diff
@@ -0,0 +1,90 @@
+
+
+# Import your own code
+
+## Configure services
+
+In the previous step, you created a collection of empty configuration files that have given your project the following structure:
+    
+```.
+├── .platform
+│   ├── routes.yaml
+│   └── services.yaml
+├── .platform.app.yaml
+└── < application code >
+```
+
```

Comment:
A NodeJS example might be nice here, too. 

Ditto for routes.yaml and .platform.app.yaml